src/lib.rs

```diff
@@ -96,6 +96,27 @@ extern crate addr2line;
           unix,
           target_os = "linux"))]
 extern crate findshlibs;
+#[cfg(all(feature = "gimli-symbolize",
+          unix,
+          target_os = "linux"))]
+extern crate gimli;
+#[cfg(all(feature = "gimli-symbolize",
+          unix,
+          target_os = "linux"))]
+extern crate memmap;
+#[cfg(all(feature = "gimli-symbolize",
+          unix,
+          target_os = "linux"))]
+extern crate object;
+#[cfg(all(feature = "gimli-symbolize",
+          unix,
+          target_os = "linux"))]
+#[macro_use]
+extern crate rental;
+#[cfg(all(feature = "gimli-symbolize",
+          unix,
+          target_os = "linux"))]
+extern crate stable_deref_trait;
```

Comment:
Does this crate already depend on `cfg-if`? If so, maybe we could do

```rust
cfg_if! {
    if #[cfg(all(feature = "gimli-symbolize", unix, target_os = "linux"))] {
        extern crate addr2line;
        extern crate findshlibs;
        extern crate gimli;
        extern crate memmap;
        extern crate object;
        extern crate rental;
        extern crate stable_deref_trait;
    } else {
        // nothing here, but cfg-if! requires an else
    }
}
```

Which is less `#[cfg(..)]` noise.

Reply:
Done, but without the else because cfg-if doesn't require it anymore.